src/app/api/_api-services/onchain_db_service/subsquid_service/subsquidUtils.ts

```
@@ -219,4 +381,125 @@ export class SubsquidUtils extends SubsquidQueries {
+			support: '0',
+			timeSplitVotes: [],
+			votesByConviction: [],
+			delegationVotesByConviction: []
+		};
+
+		let totalAye = BN_ZERO;
+		let totalNay = BN_ZERO;
+
+		votes.forEach((vote) => {
+			const balance = new BN(vote.balance?.value || vote.balance.abstain || '0');
+			const voteBalance = type === EAnalyticsType.CONVICTIONS ? this.getVotingPower(balance.toString(), vote.lockPeriod) : balance;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)